packages/test-base/src/commonMain/kotlin/io/realm/kotlin/test/util/TypeDescriptor.kt

```diff
@@ -336,6 +363,11 @@ object TypeDescriptor {
         elementTypesForCollection(elementClassifiers, CollectionType.RLM_COLLECTION_TYPE_SET)
     val elementTypesForDictionary =
         elementTypesForCollection(elementClassifiers, CollectionType.RLM_COLLECTION_TYPE_DICTIONARY)
+    val elementTypesForSerializers = elementClassifiers.mapNotNull { classifier: KClassifier ->
```

Comment:
I don't really get why we need to have a specific property for these. Shouldn't we just support serialization of all the elements classifiers that we support in our models?